// generic-search-list-lib/esm2020/lib/generic-search-list-lib.module.mjs
import { NgModule } from '@angular/core';
import { GenericSearchListLibComponent } from './generic-search-list-lib.component';
import * as i0 from "@angular/core";
export class GenericSearchListLibModule {
}
GenericSearchListLibModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.6", ngImport: i0, type: GenericSearchListLibModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GenericSearchListLibModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.6", ngImport: i0, type: GenericSearchListLibModule, declarations: [GenericSearchListLibComponent], exports: [GenericSearchListLibComponent] });
GenericSearchListLibModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.6", ngImport: i0, type: GenericSearchListLibModule, imports: [[]] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.6", ngImport: i0, type: GenericSearchListLibModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [
                        GenericSearchListLibComponent
                    ],
                    imports: [],
                    exports: [
                        GenericSearchListLibComponent
                    ]
                }]
        }] });
